src/criticality.rs

```diff
@@ -0,0 +1,55 @@
+use std;
```

Comment:
Is there a particular reason to move this to its own module? I think it's not a bad change, but would be great if you could ellaborate. Also, some module-level documentation would be appreciated.

Reply:
I did not change the actual code, and it seemed that `std` was needed as an include, so I did that.